src/main/java/companySetUp/ConfigurationBillingProcessPage.java

```diff
@@ -109,7 +109,7 @@ public void clickLoginButton() {
 		clickLoginButton.click();
 	}
 
-	@FindBy(how = How.XPATH, using = "//*[@id='menu.link.configuration']/div/div/ul/a[5]/li/span")
```

Comment:
@mayankchhajer TODO - Cannot remove ID attribute unless it is a mistake or incorrect value.